docs/docs-content/registries-and-packs/deploy-pack.md

```diff
@@ -271,238 +288,512 @@ Go ahead and review each of the following five files in the pack.
-<br />
+Next, make the registry server accessible to the public using [Ngrok](https://ngrok.com/) reverse proxy so that you can configure it later in Palette. Execute the command below to expose the registry server listening on port 5000 via an HTTP tunnel. 
+
 
 ```bash
 ngrok http 5000 --log-level debug
 ```
 
-The command above will reserve the current bash session and display the status of each HTTP request made to the Ngrok server later in this tutorial. The screenshot below shows the registry server successfully exposed via Ngrok.  
+This command reserves the current bash session and displays the status of each HTTP request made to the Ngrok server. The image below shows the registry server successfully exposed via Ngrok.  
 
-<br />
 
 ![Screenshot of registry server exposed via ngrok](/tutorials/deploy-pack/registries-and-packs_deploy-pack_ngrok-start.png )
 
+
+Check if the registry server is accessible from outside the tutorial container by visiting the `/health` endpoint. Open your browser and go to *https://Your-URL-Here/health*, replacing the base URL with the Ngrok URL output. You should get a `{"status":"UP"}` response.
+
+</TabItem>
+
+<TabItem label="ECR" value="ECR_Registry">
+
+<br />
+
+An Amazon ECR private registry is provided to each AWS account, allowing the creation of repositories within it.
+
+The initial step to creating the pack's repository in the ECR registry is to export your AWS credentials as environment variables for authentication. 
+
+In the tutorial container bash session, export the following variables. This tutorial utilizes **us-east-1** as the default region.
+
+```bash
+export AWS_ACCESS_KEY_ID=<Your_Access_Key_ID>
+export AWS_SECRET_ACCESS_KEY=<Your_Secret_Access_Key>
+export AWS_DEFAULT_REGION=<Your_Default_Region>
+```
+
+Next, export the variables below, which you will use later to create the ECR repository and push the pack.
+
+- `REGISTRY_NAME` - the name of your ECR registry. This tutorial uses **spectro-oci-registry**.
+- `NAME` - the pack's name, which must match the name in the **pack.json** file. 
+- `VERSION` - the pack's version, which must match the version in the **pack.json** file.
+- `ACCOUNT_ID` - your AWS account ID, containing only numerical digits and no dashes.
+
+```bash
+export REGISTRY_NAME=spectro-oci-registry
+export NAME=hellouniverse
+export VERSION=1.0.0
+export ACCOUNT_ID=<Your_AWS_Account_ID>
+```
+
+Create a base path repository to store your pack repositories using the AWS CLI, which is already installed in the tutorial container. Follow the provided structure below.
+
+```bash
+aws ecr create-repository --repository-name $REGISTRY_NAME/spectro-packs/archive --region $AWS_DEFAULT_REGION
+```
+
+Next, create the repository to store the Hello Universe pack.
+
```

Comment:
Maybe we should add a callout box explaining the path `spectro-packs/archive`, that seems to be important.

Reply:
Done.